.github/bin/upload-checksums-file

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+set -e
+
+build_tag="$(echo "${REF}" | cut -d'/' -f3)"
+
+# Download every release asset
+download_dir="releases/${build_tag}"
+gh release download "${build_tag}" --dir "${download_dir}"
+
+# Write checksums file
+cd "${download_dir}" || exit
+checksums_file="configlet_${build_tag}_checksums_sha256.txt"
```

Comment:
I'm using underscores here (despite the current naming convention using hyphens) because:
- We need to change the naming convention in the future to support extra platforms.
- Semantic versioning allows only hyphens for naming prerelease versions, and using underscores everywhere else helps indicate that.
- I think it's more readable.

One project that uses underscores for release assets is https://github.com/cli/cli/releases

See https://github.com/exercism/configlet/issues/363.

Reply:
Those seem like reasonable arguments.